TESTING.md

```diff
@@ -103,3 +105,34 @@ Alternatively, you can start the integration tests immediately without the need
 $ yarn itest:preview
 ```
 This will automatically start a Mirage dev server, run the integration tests on that server, and tear down the server on completion.
+
+### Tips
+* Running the integration tests will open a Firefox browser and simulate any actions that you instruct the browser to perform. That means we must first navigate to the local Cryostat Web page, before performing any useful testing.
+* Let's get started! In our `beforeAll` jest declaration, we setup our web driver with our default configuration (found in [src/itest/util.ts](src/itest/util.ts)), and then use that driver to create our first **Page Object**. A Page Object is an abstraction that acts as an interface to your web pages. For more info on the **Page Object Model** in Selenium: https://www.selenium.dev/documentation/test_practices/encouraged/page_object_models/.
+```ts
+ beforeAll(async function () {
+    driver = await setupDriver();
+    cryostat = Cryostat.getInstance(driver);
+    dashboard = await cryostat.navigateToDashboard();
+
+    await cryostat.skipTour();
+    await cryostat.selectFakeTarget();
+  });
+```
+In the previous example, we created the Cryostat top level Page Object (PO) by calling `Cryostat.getInstance(driver)` and using the driver we created before.
+We then use call a function `navigateToDashboard` to both allow the Cryostat PO to tell the web browser to navigate to the `Dashboard` page, and also obtain another `Dashboard` Page Object. The we can call more functions on our `Dashboard` object in order to simulate more browser actions with our mirage testing server. Add more methods to each PO, to test more actions. The point is, we want to abstract each browser action, so that even if something changes within our code, (e.g. a class tag is renamed, or a button is placed in a different component), all we have to do is change the underlying implementation of each function to keep tests consistent.
```

Comment:
```suggestion
We then use call the function `navigateToDashboard` to allow the Cryostat PO to tell the web browser to navigate to the `Dashboard` page, and obtain a `Dashboard` Page Object. Then, we can call more functions on our `Dashboard` object in order to simulate more browser actions.

Add more methods to each PO, to test more actions. The point is, we want to abstract each browser action, so that even if something changes within our code, (e.g. a class tag is renamed, or a button is placed in a different component), all we need is to change the underlying implementation of each function to keep tests consistent.
```